drivers/misc/ethos_u/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright 2022 Arm Limited and/or its affiliates <open-source-office@arm.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
```

Comment:
Again: this commit is doing more things other than adding a module. Please split changes accordingly.

Reply:
Fixed